src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -56,9 +64,15 @@ provideEditingConfig(Homepage, {
         'sitePortalOverviewPage',
         'siteSearchResultsPage',
         'siteUserProfilePage',
+        'siteDropShadow',
+        'siteRoundedCorners',
```

Comment:
* [ ] FYI These are a good candidates for a dedicated style/appearance tab.

Reply:
I might do this with the next PR, that will introduce custom colors.